tests/v2/testlib/wakunode.nim

```diff
@@ -0,0 +1,73 @@
+import
+  std/options,
+  stew/results,
+  stew/shims/net,
+  chronos,
+  libp2p/switch,
+  libp2p/builders,
+  libp2p/nameresolving/nameresolver,
+  libp2p/crypto/crypto as libp2p_keys,
+  eth/keys as eth_keys
+import
+  ../../../waku/v2/waku_node,
+  ../../../waku/v2/node/peer_manager,
+  ../../../waku/v2/protocol/waku_enr,
+  ../../../waku/v2/protocol/waku_discv5,
+  ./common
+
+
+# Waku node
+
+proc newTestWakuNode*(nodeKey: crypto.PrivateKey,
```

Comment:
mind in another PR adding some default?

so for exameple the boilerplate widely used:
newTestWakuNode(generateSecp256k1Key(), ValidIpAddress.init("0.0.0.0"), Port(0))

becomes just newTestWakuNode, since most of the times we are using 0.0.0.0, port 0 and a random key?

Reply:
Makes sense. Feel free to open a PR and add it :)